cipherstash-pg.gemspec

```diff
@@ -25,6 +25,9 @@ Gem::Specification.new do |spec|
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features|vendor)/}) }
   end
+  spec.files << "vendor/driver/database-extensions/postgresql/install.sql"
+  spec.files << "vendor/driver/database-extensions/postgresql/uninstall.sql"
```

Comment:
A quick note for other reviewers: `git ls-files` doesn't include files that sit inside a git submodule, even if that submodule is checked out.